R/nuts_de.R

```diff
@@ -0,0 +1,8 @@
+#' Correspondence table between German postcodes and NUTS level 3 codes
+#'
+#' @source "© European Union - GISCO, 2021, postal code point dataset,
+#' Licence CC-BY-SA 4.0 available at https://ec.europa.eu/eurostat/web/gisco/geodata/reference-data."
+#' @examples
+#' nuts_de
+#' @keywords internal
+"nuts_de"
```

Comment:
Congrats! This little file was so much work to get right -- particularly the license.